### PR TITLE
✨ Cards, Selectbox & Navigation

### PR DESCRIPTION
**Pull request by Matt-1985**
Finished my main elements with styled components. 

please note: These are prototypes and not styled according to my UI yet

### CODE REVIEW

File: client/src/components/DetailCard.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import styled from "styled-components/macro";
+import restaurant from "../stories/assets/restaurant.png";
+
+const FinalCardContainer = styled.div`
```
Comment (kschmitz123) [inline client/src/components/DetailCard.js:5]: these components look the same as the decision components. you might wanna add a "general" component instead

File: client/src/components/Card.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components/macro";
+import restaurant from "../stories/assets/restaurant.png";
```
Comment (kschmitz123) [inline client/src/components/Card.js:3]: i would move the assets folder to src

**Review by kschmitz123 — COMMENTED** [review]
lgtm 👍

File: client/src/components/DetailCard.js (hunk 1 above)
Comment (Matt-1985) [reply client/src/components/DetailCard.js:5]: Thank you! would you explain which component you referring to? I can't figure out which one you mean

File: client/src/components/Card.js (hunk 2 above)
Comment (Matt-1985) [reply client/src/components/Card.js:3]: Good idea, added to my to do list now! 🦾 i'll do it when i refactore my code, thank you!

**Pull request merged**